content/en/blog/8-must-know-serverless-terms.md

```diff
@@ -0,0 +1,158 @@
++++
+title = "8 Must Know Serverless Terms For A Developer"
+date = "2022-03-01T11:30:34+05:30"
+author = "Atulpriya Sharma"
+description = "Planning to start your cloud native journey? Here are 8 Must Know Serverless Terms For A Developer."
+categories = ["General"]
+type = "blog"
+images = ["/images/featured/serverless-terms.png"]
++++
+
+I’ve been a mobile developer for most of my professional career.
+The transition from Java to Android wasn’t a big deal. However, over the years the entire technological landscape has transformed, thanks to serverless and microservices. Many of the mobile apps that I developed have a serverless component running somewhere on the cloud.
```

Comment:
These statements doesnt fit in the context.

The transition from Java to Android wasn’t a big deal. However, over the years the entire technological landscape has transformed, thanks to serverless and microservices.

May be something of this sort

I’ve been a mobile developer for most of my professional career. Many mobile application has shifted their backend from microservices to serverless. As I focused on building serverless applications, this post highlights terms every serverless developer should know.